### PR TITLE
add missing optional parameters to API v2 project creation endpoint

### PR DESCRIPTION
**Pull request by decathorpe**
Fixes: #1463

### CODE REVIEW

File: anitya/api_v2.py [hunk 1]
```diff
@@ -471,7 +475,11 @@ def post(self):
             "homepage": fields.Str(required=True),
             "backend": fields.Str(required=True),
             "version_url": fields.Str(missing=None),
+            "version_scheme": fields.Str(missing=None),
```
Comment (Zlopez) [inline anitya/api_v2.py:478]:
```suggestion
            "version_scheme": fields.Str(missing="RPM"),
```
`version_scheme` should be always set, it's not required field in the API, but it's set to RPM in `create_project` function if not provided. Let's set it to "RPM" if it's not provided.

File: anitya/api_v2.py [hunk 2]
```diff
@@ -449,8 +449,12 @@ def post(self):
         :reqjson string backend: The project backend (github, folder, etc.).
         :reqjson string version_url: The URL to fetch when determining the
                                      project version (defaults to null).
+        :reqjson string version_scheme: The project version scheme.
```
Comment (Zlopez) [inline anitya/api_v2.py:452]:
```suggestion
        :reqjson string version_scheme: The project version scheme. If missing, it's set to "RPM".
```

**Review by Zlopez — CHANGES_REQUESTED** [review]
One thing that should be changed, otherwise looks fine.

File: anitya/api_v2.py (hunk 1 above)
Comment (decathorpe) [reply anitya/api_v2.py:478]: Makes sense! Thanks for the review.

**Review by Zlopez — APPROVED** [review]

**Comment by decathorpe** [comment]
Awesome, thanks for merging!

**Pull request merged**